kornia/augmentation/container/utils.py

```diff
@@ -337,6 +337,11 @@ class BBoxApplyInverse(ApplyInverseImpl):
     def _get_padding_size(cls, module: Module, param: Optional[ParamItem]) -> Optional[Tensor]:
         if isinstance(module, RandomCrop) and param is not None and isinstance(param.data, dict):
             return param.data["padding_size"]
+        elif param is not None and isinstance(param.data, list):
+            padding_size = None
+            for param_ in param.data:
+                padding_size = param_.data.get('padding_size', None)
```

Comment:
I have no idea what this will impact... but I think it would be the same as this here

```suggestion
    padding_size = param.data[-1].data.get('padding_size', None)
```

but shouldn't it be the last pad size found instead of the last or None?
```python
return next((p_.data['padding_size'] for p_ in param.data[::-1] if 'padding_size' in p_.data), None)
```

Reply:
Your first suggestion assumes you know where in the list (last entry) the random crop params can be. They could be in another position.

With the last comment you mean it should be the _first_ pad size found instead of last? If so, sure, it maybe makes more sense, but it will also fail if there is more than one anyway.

Reply:
The first case does the same thing as your code -- which I think is the problem

The second get the last because of the reverse list `param.data[::-1]`

Reply:
ah yes totally! thanks

Reply:
Anyway, going in this direction is still not a good solution, I wouldn't like to merge this since we know it will eventually fail. If you have any ideas on how to cover the more general case I'd be happy to discuss them.

Reply:
I'll check throughout the day to see if I have any ideas 😅

Reply:
cc @shijianjian any idea for this case?